frontend/lib/norent/tenant-info-page.tsx

```diff
@@ -73,7 +73,7 @@ export const NorentTenantInfoPage: React.FC<ProgressStepProps> = (props) => {
       <ClearSessionButton
         to={NorentRoutes.locale.home}
         portalRef={cancelControlRef}
-        label="Cancel letter"
+        label="Cancel"
```

Comment:
Changed this label just to test out button spacing! Will make sure all buttons like these say "Cancel" once we actually build out the front end for all of the letter builder steps.